src/Entity/AamcResourceType.php

```diff
@@ -22,6 +23,7 @@ class AamcResourceType implements AamcResourceTypeInterface
     use IdentifiableStringEntity;
     use TitledEntity;
     use CategorizableEntity;
+    use DescribableEntity;
```

Comment:
nice!

Reply:
I spent quite a while in our git history trying to figure out why we didn't use this initially. Meant to ask you as well, but forgot about it. I can see no reason not to so I did. Some of these on Objective entities as well.